### DISCUSSION TITLE
add a warning that every single script within /config/extended/scripts/ is overwritten at initialization.

### DISCUSSION THREAD

**Discussion by basings** [RandomNinjaAtk/docker-lidarr-extended#197] [Ideas]
gently said, it was a big surprise that every change was overwritten once I restarted the container. I don't want anyone to be as surprised as I was.

https://github.com/RandomNinjaAtk/docker-lidarr-extended/blob/main/root/etc/cont-init.d/98-script-setup.bash

> rm -rf /config/extended/scripts/*

**Comment by RandomNinjaAtk** [comment]
Adding a warning is probably not a bad idea, the question would be where to put it... Any thoughts?

Also, if your making changes, what is being changed? Might be useful to add it to the included scripts... contributions welcome, as long as they make sense... Just some thoughts...

**Reply by basings** [reply]
I'm still reading through the source code but it looks like the script always deletes everything, it always assumes that there was some sorcery going on and hence it deletes everything and recreates everything. It's not only the case for the scripts, all the folders as well, always deleted and recreated, etc. I suppose that's there for a reason but you can't change anything manually and permanently within the container if I'm understanding it right. Maybe a short entry in the [wiki](https://github.com/RandomNinjaAtk/docker-lidarr-extended/wiki) is which says something like

> Every file within the container is always recreated and overwritten, do not manually edit the files. If you want to edit the scripts, edit the original files in $pathtooriginalfiles and restart the container.

**Reply by RandomNinjaAtk** [reply]
Not every file is purged/recreated. Only temp directories are constantly purged and the /config/extended/scripts folder...

The reason for purging the script folders is to ensure modifications aren't made and the user is running the latest version. Because if it didn't do that, troubleshooting problems would be a nightmare because there would be to many variables to account for... There also shouldn't be a reason to modify it, if there is, then the user should consider opening an issue/request or PR with fixes...

If the script works one time, it should work every-time...
